modules/client-configuration/pages/cobbler.adoc

```diff
@@ -126,7 +124,7 @@ virsh net-dumpxml --inactive network > network.xml
 ----
 virsh net-define network.xml
 ----
-+
+
```

Comment:
```suggestion
+
```
adoc needs the plus to make the following sentence part of the last step of the procedure.